redwood-lazylayout-compose/src/commonMain/kotlin/app/cash/redwood/lazylayout/compose/LazyItemScope.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.lazylayout.compose
+
+import androidx.compose.runtime.Stable
+import app.cash.redwood.LayoutScopeMarker
+import app.cash.redwood.Modifier
+import app.cash.redwood.layout.api.CrossAxisAlignment
+
+@LayoutScopeMarker
+public interface LazyItemScope {
+  @Stable
+  public fun Modifier.alignment(alignment: CrossAxisAlignment): Modifier =
+    then(AlignmentImpl(alignment))
```

Comment:
@colinrtwhite Do you have any ideas on how to convert this to `horizontalAlignment`/`verticalAlignment` without introducing a new scope? `LazyItemScope` is used for more than just scoping modifiers (see `LazyListIntervalContent`), so I'm a bit hesitant to just have two separate scopes for `LazyRow` and `LazyColumn`.